atc: propagate span context into build containers (#5653)
concourse/concourse: release-notes/latest.md

* When [distributed tracing](https://concourse-ci.org/tracing.html) is configured, Concourse will now emit spans for several of its backend operations, including resource scanning, check execution, and job scheduling. These spans will be appropriately linked when viewed in a tracing tool like Jaeger, allowing operators to better observe the events that occur between resource checking and build execution. #5659

#### <sub><sup><a name="5653" href="#5653">:link:</a></sup></sub> feature

> * When [distributed tracing](https://concourse-ci.org/tracing.html) is configured, all check, get, put, and task containers will be run with the `TRACEPARENT` environment variable, which contains information of the parent span following the [w3c recommendation format](https://www.w3.org/TR/trace-context/#traceparent-header):

Reviewer: ```suggestion
* When [distributed tracing](https://concourse-ci.org/tracing.html) is configured, all check, get, put, and task containers will be run with the `TRACEPARENT` environment variable set, which contains information about the parent span following the [w3c trace context format](https://www.w3.org/TR/trace-context/#traceparent-header):
```
